specification/apimanagement/resource-manager/readme.azureresourceschema.md

```diff
@@ -51,12 +51,14 @@ input-file:
   - Microsoft.ApiManagement/preview/2020-06-01-preview/apimopenidconnectproviders.json
   - Microsoft.ApiManagement/preview/2020-06-01-preview/apimpolicies.json
   - Microsoft.ApiManagement/preview/2020-06-01-preview/apimpolicydescriptions.json
+  - Microsoft.ApiManagement/preview/2020-06-01-preview/apimportalrevisions.json
```

Comment:
Are these two files part of the reverted change?

Reply:
nope, it's new swagger file, and should be added to the readme file.